meta-microservicebus-node/recipes-microservicebus-node/nodejs/nodejs_14.18.1.bb

```diff
@@ -0,0 +1,200 @@
+DESCRIPTION = "nodeJS Evented I/O for V8 JavaScript"
```

Comment:
Better to use nodejs 16.14.2 as is delivered in Kirkstone and solve the issue in msb-core instead.
msb-core issue [91](https://github.com/axians/microservicebus-core/issues/91)

Reply:
I agree.

Should I remove this nodejs v14 recipe and `PREFERRED_VERSION:nodejs ?= "14.18.1"` from layer.conf or do you want to keep it until https://github.com/axians/microservicebus-core/issues/91 is implemented?

I added it for convenience but can also move it to our app layer if you prefer to not have it here.